.github/workflows/manual-testing.yml

```diff
@@ -0,0 +1,148 @@
+name: Manual Testing
+on:
+  workflow_dispatch:
+    inputs:
+      networkNodeTag:
+        description: 'Specify desired Network Node image tag'
+        required: true
+        default: ''
+      mirrorNodeTag:
+        description: 'Specify desired Mirror-Node image tag'
+        required: true
+        default: ''
+      relayTag:
+        description: 'Specify desired Hedera JSON-RPC Relay tag'
+        required: true
+        default: ''
+run-name: Manual Test with N:${{ inputs.networkNodeTag }}, M:${{ inputs.mirrorNodeTag }} and R:${{ inputs.relayTag }}
+permissions:
```

Comment:
Unfortunately it fails otherwise with:

```
Invalid workflow file:  .github/workflows/manual-testing.yml#L19 The workflow is not valid. 
.github/workflows/manual-testing.yml (Line: 19, Col: 3): Error calling workflow '/hedera-smart-contracts/.github/workflows/test-workflow.yml@a7e84d8bd1209ac5886ee6f836d53f7d32333bfb'. 
The workflow is requesting 'checks: write, contents: write, pull-requests: write', but is only allowed 'checks: none, contents: read, pull-requests: none'.
```

More info can be found [here](https://www.kenmuse.com/blog/github-actions-workflow-permissions/).